src/backend/executor/execMain.c

```diff
@@ -648,6 +650,18 @@ ExecCheckOneRelPerms(RTEPermissionInfo *perminfo)
 	requiredPerms = perminfo->requiredPerms;
 	Assert(requiredPerms != 0);
 
+	/* 
+	 * Some extension may want to avoid permission check for some special
```

Comment:
Let us make comment specific to babelfish, this check no longer applies to anything outside BBF